app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -1,7 +1,33 @@
 (function (GOVUK) {
   GOVUK.GDM.analytics.pageViews = {
     'init': function () {
+      this.setCustomDimensions()
       GOVUK.analytics.trackPageview();
-    }
+    },
+
+    'setCustomDimensions': function() {
+      var customDimensionData,
+          dimensionValToBeSet,
+          h1Val;
+      
+      if (this.createOpportunityResponseRegex.test(GOVUK.GDM.analytics.location.pathname())) {
+        customDimensionData = $('[data-analytics=customDimension]').first().data('custom-dimension');
```

Comment:
It feels like we should be setting another attribute for the dimension index here.

I'm basing this on the assumption that we're following the precedent of the `data-analytics` attribute marking the type of analytics information we set with `data-analytics="trackPageView"`. If that's the case, we need to follow that convention of all the information needed for the analytics event being available in the data attributes on the node. At the moment, this assumes a custom dimension on this page will always be set on index 25.

Alternatively, this could be rewritten to be more specific to our case, for example:

```
<div class="grid-row" data-analytics="supplierNotEligible" data-reason="{{ ineligibility_reason }}">
```